classes/05class/exercises/c05-3m01/starkmatt/docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: '3.4'
+
+services:
+
+ aws:
+     build: .
+     env_file: .env
+     working_dir: /code
+     volumes:
+     - .:/code
```

Comment:
Hey Matt, the exercise looks good. 

One tip is that here in your `docker-compose.yaml` you could have mapped the `~/.aws/` folder to the container so the same credentials available in the host would be available in the container. It is handy when you are using `profiles` which is a good practice when using AWS CLI.

Check this answer: https://github.com/devopsacademyau/academy/pull/581/files#diff-04bca59852327900204639d576d0e276R8